week-notes/2020-01-24.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 2020-01-24
+parent: Week Notes
+---
+# BC Common Components (CoCo) Weeknote 2:
+> ## 2020-01-24
+___
+
+This is the second post in a series of weeknotes recapping what the BC Common Components (we’ve taken to calling it “CoCo”) team has done this week and what we plan to accomplish next week. All previous weeknotes are saved here. There’s also now a CoCo mission control dashboard you can track [here](https://trello.com/b/vqqXYk3l/common-components-mission-control).
+
+## Done
+- Participated in a lunch and share with the digital community to learn about what others are building and how. We had a lively conversation about Common Components and associated pain points.
+- Attended ~5 sprint reviews hosted by other teams. Highlights were the [Health Gateway team](https://github.com/bcgov/healthgateway) and [Climate Action Secretariat](https://github.com/bcgov/cas-ggircs), both exemplary teams working on impactful products and building for reuse.
+- Interviewed several teams and product owners, including GDX, Climate Action, and Mines. My highlight was an interview Terry and I had with Jill Carruthers (Mines).
```

Comment:
Remove reference to Terry and Jill. We can just pull that sentence out completely.